packages/language-support/src/syntaxValidation/semanticAnalysisWrapper.ts

```diff
@@ -63,7 +65,22 @@ export function wrappedSemanticAnalysis(
       });
     }
 
-    const semanticErrorsResult = analyzeQuery(query, 'cypher 5');
+    const validCypherVersions = ['CYPHER 5', 'CYPHER 25'];
+    let cypherVersion = 'CYPHER 5';
+    const defaultVersion = dbSchema.defaultLanguage?.toUpperCase();
+
+    if (parsedVersion && validCypherVersions.includes(parsedVersion)) {
```

Comment:
You don't need `validCypherVersions` anymore because `CypherVersion` is a type that can only be two possible values